test/system/invitations_test.rb

```diff
@@ -178,13 +179,43 @@ class InvitationDetailsTest < ApplicationSystemTestCase
       fill_in "Last Name", with: "Yamaguchi"
       click_on "Next"
 
+      # Click on next to show that bulk invitations will raise an error if not filled out properly.
```

Comment:
We can potentially put all of the testing for bulk invitations in a new file like `invitation_details_test.rb` has separate tests, but I'm okay with this for now.

Reply:
Okay, I just went ahead and put the tests in their own file, `invitation_lists_test.rb`